meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-core/initrdscripts/tegra-minimal-init_%.bbappend

```diff
@@ -0,0 +1,7 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
+
+EXTRADEPS = ""
+EXTRADEPS:tegra = "tegra-boot-tools-earlyboot"
```

Comment:
I assume we can remove the initrdscripts/tegra-minimal-init as well as I think this assumes cboot/tegra-boot-tools which also won't be a part of Jetpack 5.